manifests/kubeless.jsonnet

```diff
@@ -0,0 +1,159 @@
+local k = import "../../ksonnet.beta.1/k.libsonnet";
+local util = import "../../ksonnet.beta.1/util.libsonnet";
```

Comment:
Don't hardcode the ksonnet-lib installation location - rely on the jsonnet search path.

Change these imports to just `import "ksonnet.beta.1/..."` and use `jsonnet -J <ksonnet-lib dir>` in examples/docs/scripts.  `kubecfg` (if you're using that) supports `-J` and also a `KUBECFG_JPATH` env var for the search path.